pkg/container/vector/tools.go

```diff
@@ -490,7 +522,7 @@ func (v *Vector) CompareAndCheckAnyResultIsTrue(ctx context.Context, vec *Vector
 	default:
 		return false, moerr.NewInternalErrorNoCtx("unsupport compare type")
 	}
-
+	//TODO: T_embedding won't be used in zonemap.
```

Comment:
I think dn can not perceive this stuff at all. For dn it's a varchar and they handle it how they originally did. . zonemap is fine even if it's built.